discord/guild.py

```diff
@@ -464,7 +469,7 @@ def _add_role(self, role: Role, /) -> None:
         self._roles[role.id] = role
 
     def _remove_role(self, role_id: int, /) -> Role:
-        # this raises KeyError if it fails..
+        # this raises KeyError if it fails...
```

Comment:
I think it's more likely that this was meant to be a single period, not an ellipsis